CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Added
 
 - `generate_random_felt()` for generating (pseudo) random felt value.
+- Printing information about compiling Sierra files by `universal-sierra-compiler` before tests execution
```

Comment:
```suggestion
- Printing information about compiling Sierra files by `universal-sierra-compiler`
```

It can also happen during tests execution right?

Reply:
Correct, forgot to update this